### PR TITLE
Remove snapshot

### PR DESCRIPTION
**Pull request by CuriousVini**
Remove snapshot versions from cdap

### CODE REVIEW

File: cdap-docs/_common/common_conf.py [hunk 1]
```diff
@@ -327,7 +327,7 @@ def get_git_build_vars():
 """ % version_tuple
 
 if release:
-    # Create release range, such as [4.0.0-SNAPSHOT,4.1.0):
+    # Create release range, such as [4.0.0,4.1.0):
```
Comment (awholegunch) [inline cdap-docs/_common/common_conf.py]: don't change this comment.

File: cdap-ui/app/tracker/services/my-jump-factory.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ function myJumpFactory($state, myTrackerApi, $rootScope) {
   let corePluginsArtifacts = {
     name: 'core-plugins',
     scope: 'SYSTEM',
-    version: '1.5.0-SNAPSHOT'
```
Comment (awholegunch) [inline cdap-ui/app/tracker/services/my-jump-factory.js:29]: I don't know about this one: check with @ajainarayanan

File: cdap-ui/server/config/cdap-ui-config.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "tracker": {
     "artifact": {
       "name": "tracker",
-      "version": "0.3.0-SNAPSHOT",
+      "version": "0.3.0",
```
Comment (awholegunch) [inline cdap-ui/server/config/cdap-ui-config.json:5]: This is assuming that tracker has been bumped.

File: cdap-ui/server/config/cdap-ui-config.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
   "navigator": {
     "artifact": {
       "name": "navigator",
-      "version": "0.4.0-SNAPSHOT",
+      "version": "0.4.0",
```
Comment (awholegunch) [inline cdap-ui/server/config/cdap-ui-config.json:15]: ibid.

**Comment by awholegunch** [comment]
Aside from one comment about a comment that shouldn't be changed, the rest of the ``4.0.0-SNAPSHOT`` changes LGTM. Some of the others need to be checked or coordinated with the other repos.

File: cdap-ui/app/tracker/services/my-jump-factory.js (hunk 2 above)
Comment (anwar6953) [reply cdap-ui/app/tracker/services/my-jump-factory.js:29]: vinisha's change looks appropriate.

File: cdap-docs/_common/common_conf.py (hunk 1 above)
Comment (anwar6953) [reply cdap-docs/_common/common_conf.py]: +1 to jj's comment. This seems like just an example in a comment.

**Comment by anwar6953** [comment]
No concerns besides @awholegunch's.
👍

File: cdap-ui/app/tracker/services/my-jump-factory.js (hunk 2 above)
Comment (elfenheart) [reply cdap-ui/app/tracker/services/my-jump-factory.js:29]: This is correct

File: cdap-ui/server/config/cdap-ui-config.json (hunk 3 above)
Comment (elfenheart) [reply cdap-ui/server/config/cdap-ui-config.json:5]: It has

File: cdap-ui/server/config/cdap-ui-config.json (hunk 4 above)
Comment (elfenheart) [reply cdap-ui/server/config/cdap-ui-config.json:15]: This too

**Comment by elfenheart** [comment]
UI :+1: LGTM

**Pull request merged**